packages/components/src/RichTitle/RichTitle.component.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import HeaderTitle from '../HeaderTitle';
+import { Action, Actions } from '../Actions';
+import Inject from '../Inject';
+
+export default function RichTitle(props) {
+	const Renderers = Inject.getAll(props.getComponent, {
+		Actions,
+	});
+
+	return [<HeaderTitle title={props.title} />, <Renderers.Actions actions={props.right} />];
```

Comment:
you need a key props here.
So a richTitle is a HeaderTitle + an Action. This seems to fit the old guideline. 
I guess we need to talk here.